docs-devsite/vertexai-preview.md

```diff
@@ -36,12 +43,12 @@ The Vertex AI in Firebase Web SDK.
 |  [BlockReason](./vertexai-preview.md#blockreason) | Reason that a prompt was blocked. |
 |  [FinishReason](./vertexai-preview.md#finishreason) | Reason that a candidate finished. |
 |  [FunctionCallingMode](./vertexai-preview.md#functioncallingmode) |  |
-|  [FunctionDeclarationSchemaType](./vertexai-preview.md#functiondeclarationschematype) | Contains the list of OpenAPI data types as defined by https://swagger.io/docs/specification/data-models/data-types/ |
 |  [HarmBlockMethod](./vertexai-preview.md#harmblockmethod) |  |
 |  [HarmBlockThreshold](./vertexai-preview.md#harmblockthreshold) | Threshold above which a prompt or candidate will be blocked. |
 |  [HarmCategory](./vertexai-preview.md#harmcategory) | Harm categories that would cause prompts or candidates to be blocked. |
 |  [HarmProbability](./vertexai-preview.md#harmprobability) | Probability that a prompt or candidate matches a harm category. |
 |  [HarmSeverity](./vertexai-preview.md#harmseverity) | Harm severity levels. |
+|  [SchemaType](./vertexai-preview.md#schematype) | Contains the list of OpenAPI data types as defined by https://swagger.io/docs/specification/data-models/data-types/ |
```

Comment:
NOT BLOCKING

Usually, we try to make all URLs into hyperlinks rather than having the actual URL exposed.

(comment applies to several other instances, even in other files)

Reply:
Fixed.

Reply:
I can't seem to find the corresponding new file in the new set of files?